### ISSUE TITLE
Scheduled analyzers

### ISSUE THREAD

**Issue by MinhNamNguyen** [TheHive-Project/TheHive#1091]
### Request Type
Feature Request

### Work Environment

| Question              | Answer
|---------------------------|--------------------
| OS version (server)       | Ubuntu
| OS version (client)       | Ubuntu
| TheHive version / git hash   | 3.3.0-1
| Package Type              | Docker
| Browser type & version    | Chrome

### Problem Description
The ability to scheduled some tasks, like launching an analyzer

**Comment by veeral-patel** [comment]
Hey @MinhNamNguyen -- just curious, what's your use case for this?

**Comment by MinhNamNguyen** [comment]
Hi, I have developed some scripts that scan the intranet and some shares and i would like to schedule it every week or so

**Comment by veeral-patel** [comment]
@MinhNamNguyen so you'd like to create a new case every week, on a schedule, that launches an analyzer?

Or would you like to run an analyzer every week in one case?

**Comment by MinhNamNguyen** [comment]
More likely run an analyzer every week in a single case

**Comment by veeral-patel** [comment]
@MinhNamNguyen have you tried doing this with the API?

**Comment by veeral-patel** [comment]
@MinhNamNguyen if you don't mind, could you share a little more information about your script and the case you're running it in?

Maybe that way we can make the UX even better :)

**Comment by MinhNamNguyen** [comment]
I didn't try using TheHive4Py, i will take a look at it
My script is about looking for some patterns and regex in some Windows Shares, i use it to search for leaks

**Comment by MinhNamNguyen** [comment]
But there will be other script than mine so it would be better if there was a tool that allow to schedule them

**Comment by veeral-patel** [comment]
@MinhNamNguyen So this seems like an interesting idea, I wonder what other use cases there are :) 

I'd be willing to get on a 30 minute call with you sometime to chat so we can run through a couple ways I could implement this:

1. TheHive letting you schedule analyzers in the UI
2. Me writing documentation on how to schedule a custom TheHive4Py-based script using a cronjob

Feel free to email me at veeral.patel@berkeley.edu and we can get the ball rolling!

**Comment by dumar** [comment]
Your use case doesn't sound like an analyzer. Analyzers usually act on some input data. Its more like an independent job that could be scheduled using cron and based on the results you could create case using API.

**Comment by MinhNamNguyen** [comment]
True, but i use thehive and cortex as an interface that allow me to control tools that i create, and also use the analyzers already developped
